src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -169,7 +169,7 @@ exports.main = async function (event, context) {
         // return updated/modified config with latest ETag
         return {
           statusCode: 200,
-          headers: { "'If-None-Match'": cachedConfig.ETag },
+          headers: { "'ETag'": cachedConfig.ETag },
```

Comment:
Should line 163 `if (event.headers['If-None-Match'] === cachedConfig.ETag) {` be updated to 
`if (event.headers['ETag'] === cachedConfig.ETag) {` ?

Reply:
no, the request header should use `If-None-Match` and the response header uses `ETag`